projects/neural_correlations/EXP1-Random/NeuCorr_Exp1.ipynb

```diff
@@ -173,79 +108,114 @@
     "negPCCX_cells = []\n",
     "negPCCY_cells = []\n",
     "\n",
+    "numSpikesX = []\n",
+    "numSpikesY = []\n",
+    "\n",
+    "numSpikes = 0\n",
+    "\n",
     "negPCCX_cols = []\n",
     "negPCCY_cols = []\n",
     "\n",
     "# Randomly generate the indices of the columns to keep track during simulation time\n",
-    "colIndicesSmall = np.random.permutation(tm.numberOfColumns())[0:4] # keep track of 4 columns\n",
-    "colIndicesLarge = np.random.permutation(tm.numberOfColumns())[0:125] # keep track of 4 columns\n",
+    "colIndicesLarge = np.random.permutation(tm.numberOfColumns())[0:125] # keep track of 125 columns = 1000 cells\n",
```

Comment:
I cannot consider all 2048 columns. This would make the calculation of the pairwise correlations very computationally expensive.